packages/core/src/List/List.spec.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { render, screen } from 'testing-library'
+import { List } from '..'
+
+describe('List', () => {
+  it('renders ordered list', () => {
+    render(
+      <List.ol color='text.light' data-testid='list'>
+        <li>Example Text 1</li>
+        <li>Example Text 2</li>
+        <li>Example Text 3</li>
+      </List.ol>
+    )
+
+    const list = screen.getByTestId('list')
+
+    expect(list).toHaveStyleRule('color', '#4f6f8f')
```

Comment:
Can you get the expected value from theme?

Reply:
Do you mean having `text.light` here instead of this hex value? I tried that and it didn't work unfortunately 😞